api/lib/domain/usecases/remove-authentication-method.js

```diff
@@ -35,16 +35,12 @@ const removeAuthenticationMethod = async function ({ userId, type, userRepositor
     );
   }
 
-  if (type === OidcIdentityProviders.POLE_EMPLOI.service.code) {
-    await _removeAuthenticationMethod(
-      userId,
-      OidcIdentityProviders.POLE_EMPLOI.service.code,
-      authenticationMethodRepository
-    );
+  if (type === OidcIdentityProviders.POLE_EMPLOI.code) {
+    await _removeAuthenticationMethod(userId, OidcIdentityProviders.POLE_EMPLOI.code, authenticationMethodRepository);
   }
 
-  if (type === OidcIdentityProviders.CNAV.service.code) {
-    await _removeAuthenticationMethod(userId, OidcIdentityProviders.CNAV.service.code, authenticationMethodRepository);
+  if (type === OidcIdentityProviders.CNAV.code) {
+    await _removeAuthenticationMethod(userId, OidcIdentityProviders.CNAV.code, authenticationMethodRepository);
   }
```

Comment:
On devrait ajouter le mot clé `return` pour éviter de faire des vérifications inutiles. On reçoit en paramètre un seul type et pas une liste donc on pourrait ajouter un `return` pour toutes les conditions précédentes.

```suggestion
  if (type === OidcIdentityProviders.CNAV.code) {
    return await _removeAuthenticationMethod(userId, OidcIdentityProviders.CNAV.code, authenticationMethodRepository);
  }
```

Reply:
Ajout de `return` pour éviter les évaluations de code inutiles fait dans un nouveau commit. Merci.